src/fields.jl

```diff
@@ -116,8 +116,8 @@ location(::Field{Lx, Ly, Lz}) where {Lx, Ly, Lz} = (Lx, Ly, Lz)
 @inline size(f::AbstractField) = size(f.grid)
 @inline length(f::AbstractField) = length(f.data)
 
-@propagate_inbounds getindex(f::AbstractField, inds...) = getindex(f.data, inds...)
-@propagate_inbounds setindex!(f::AbstractField, v, inds...) = setindex!(f.data, v, inds...)
+@inline @propagate_inbounds getindex(f::AbstractField, inds...) = getindex(f.data, inds...)
```

Comment:
I think that `@propagate_inbounds` implies `@inline`, which means that the `@inline` annotation is redundant here. Is there a demonstrated performance benefit from this change? If so, why?